hail/python/dev/requirements.txt

```diff
@@ -21,6 +21,9 @@ sphinx_rtd_theme>=1.0.0,<2
 jupyter>=1.0.0,<2
 sphinxcontrib.katex>=0.9.0,<1
 fswatch>=0.1.1,<1
+
+# benchmark requirements
+wheel>=0.41,<0.42
```

Comment:
Can you explain why you just now needed to add wheel to the dev requirements?

Reply:
I can't build the benchmark wheel when I create a new venv because `wheel` has been missing from the dev requirements. I've been too lazy to PR this fix until now lol